src/app/organization/tellers/tellers.component.html

```diff
@@ -47,14 +47,14 @@
       <ng-container matColumnDef="actions">
         <th mat-header-cell *matHeaderCellDef> Actions </th>
         <td mat-cell *matCellDef="let teller">
-          <button mat-button color="primary">
+          <button mat-button color="primary" (click)="viewCashiers($event)">
```

Comment:
```suggestion
          <button mat-button color="primary" (click)="$event.stopPropagation()">
```

I believe this would be better.

Reply:
Ok, thanks for the recommendation.